venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js

```diff
@@ -0,0 +1,128 @@
+    title: 'Medical Imaging with Deep Learning',
+    subtitle: 'MIDL 2019 Conference',
+    location: 'London',
+    date: ' 8-10 July 2019',
+    website: 'http://2019.midl.io',
+    instructions: 'Full papers contain well-validated applications or methodological developments of deep learning algorithms in medical imaging. There is no strict limit on paper length. However, we strongly recommend keeping full papers at 8 pages (excluding references and acknowledgements). An appendix section can be added if needed with additional details but must be compiled into a single pdf. The appropriateness of using pages over the recommended page length will be judged by reviewers. All accepted papers will be presented as posters with a selection of these papers will also be invited for oral presentation.<br/><br/> <p><strong>Questions or Concerns</strong></p><p>Please contact the OpenReview support team at <a href=\"mailto:info@openreview.net\">info@openreview.net</a> with any questions or concerns about the OpenReview platform.<br/>    Please contact the MIDL 2019 Program Chairs at <a href=\"mailto:program-chairs@midl.io\">program-chairs@midl.io</a> with any questions or concerns about conference administration or policy.</p><p>We are aware that some email providers inadequately filter emails coming from openreview.net as spam so please check your spam folder regularly.</p>'
+   });
+
+  Webfield.ui.spinner('#notes');
+}
+
+function renderConferenceTabs() {
+  var sections = [
+    {
+      heading: 'Full - Accept(Oral)',
+      id: 'accepted-oral-papers',
+    },
+    {
+      heading: 'Full - Accept(Poster)',
+      id: 'accepted-poster-papers',
+    },
+    {
+      heading: 'Full Submissions',
+      id: 'all-papers',
+    }
+  ];
+
+  Webfield.ui.tabPanel(sections, {
+    container: '#notes',
+    hidden: true
+  });
+}
+
+function renderContent(notes, decisionsNotes) {
+
+  var notesDict = {};
+  _.forEach(notes, function(n) {
+    notesDict[n.id] = n;
+  });
+
+  var oralDecisions = [];
+  var posterDecisions = [];
+  var submittedPapers = [];
+
+  _.forEach(decisionsNotes, function(d) {
+
+    if (_.has(notesDict, d.forum)) {
+      if (d.content.decision === 'Accept') {
+        if (d.content.presentation === 'Oral') {
+            oralDecisions.push(notesDict[d.forum]);
+        } else if (d.content.presentation === 'Poster'){
+            posterDecisions.push(notesDict[d.forum]);
+        }
+      }
+      submittedPapers.push(notesDict[d.forum]);
+    }
+  });
+
+  oralDecisions = _.sortBy(oralDecisions, function(o) { return o.id; });
+  posterDecisions = _.sortBy(posterDecisions, function(o) { return o.id; });
+  submittedPapers = _.sortBy(submittedPapers, function(o) { return o.id; });
+
+  var paperDisplayOptions = {
+    pdfLink: true,
+    replyCount: true,
+    showContents: true
+  };
+
+  Webfield.ui.searchResults(
+    oralDecisions,
+    _.assign({}, paperDisplayOptions, {showTags: false, container: '#accepted-oral-papers'})
```

Comment:
I would move `showTags: false` to the paperDisplayOptions so it doesn't need to be repeated for each tab.

Reply:
Not sure what to do about slow rendering - do you have a suggestion?

Reply:
How many total submissions are there for this conference?

Reply:
It's probably not an issues unless there are greater than 1000 submissions.

Reply:
There are under 200.

Reply:
Oh this is fine then.